index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    
+    <header class="mobile-header">
+        <a href="#">Diego Hernández</a>
+        <a href="#"><img src="./menu.png" alt="hamburger Icon" /></a>
+     
+   
+    </header>
+
+
+    <section class="content"> 
+       
+        
+        <h1>Hey There. I'm Diego</h1> 
+        <h3>I’am  web a developer.</h3> 
+        <p>I would like you to see some of my best layout work, I am able to design and build high quality websites. If you need to share any of your projects do not hesitate to contact me.</p> 
+
+      <ul class="social">
+            <li><img src="images/github.png" alt="social-icon" /></li>
+            <li><img src="images/linkedin.png" alt="social-icon" /></li>
+            <li><img src="images/angelist.png" alt="social-icon" /></li>
+            <li><img src="images/twitter.png" alt="social-icon" /></li>
+            <li><img src="images/medium.png" alt="social-icon" /></li>
```

Comment:
[ OPTIONAL ] All the icons should be linked to your social profiles. Kindly wrap all the social images in `a` tags.